packages/eslint-plugin-eds/README.md

```diff
@@ -0,0 +1,60 @@
+# @chanzuckerberg/eslint-plugin-eds
+
+Shared React [ESLint](https://eslint.org/) plugin and config for CZI's Education initiative.
```

Comment:
```suggestion
Shared React [ESLint](https://eslint.org/) plugin and config for CZI's Education Design System.
```

Reply:
Oops didn't catch these when I copy pasta'd the readme, thanks!